### PR TITLE
issue-220: Fixes ttl-writetime auto flag handling

### PR DESCRIPTION
**Pull request by arvy**
**What this PR does**:
Fixes ttl-writetime feature auto flag handling and some tests.

**Which issue(s) this PR fixes**:
Fixes #220 

**Checklist:**
- [x] Automated Tests added/updated
- [x] Documentation added/updated
- [x] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Review by faizalrub-datastax — APPROVED** [review]

File: src/resources/cdm-detailed.properties [hunk 1]
```diff
@@ -101,9 +101,17 @@ spark.cdm.connect.target.password      cassandra
 #                           columns need to be listed.
 #-----------------------------------------------------------------------------------------------------------
 spark.cdm.schema.origin.keyspaceTable                keyspace_name.table_name
+
+# Max TTL value of all non-PK columns will be used for insert on target
```
Comment (arvy) [inline src/resources/cdm-detailed.properties:105]: non-PK columns might not be specific enough - not all types support writetime/ttl (collections, counters, etc..). ..so maybe "all ttl-eligible columns" would be a better description..

Comment (pravinbhat) [reply src/resources/cdm-detailed.properties:105]: Agree. May be "all ttl-eligible non-PK columns" ?

**Review by pravinbhat — APPROVED** [review]
Looks good but it will still fail the coverage check due to build issues. Once PR #228 is merged, this should work. You will need to pull changes from main & then merge.

**Comment by msmygit** [comment]
Fixes #218

**Pull request merged**